### PR TITLE
Add SecondaryFragmentColor

### PR DESCRIPTION
**Pull request by MoochMcGee**
Since we aren't doing fragment lighting just yet, do the same thing we did with PrimaryFragmentColor

Verified working with Dragon Quest 7 warning screen.

dq7_screen_1

### CODE REVIEW

File: src/video_core/rasterizer.cpp [hunk 1]
```diff
@@ -384,9 +384,12 @@ static void ProcessTriangleInternal(const VertexShader::OutputVertex& v0,
 
                 auto GetSource = [&](Source source) -> Math::Vec4<u8> {
                     switch (source) {
-                    // TODO: What's the difference between these two?
+                    // TODO: Implement fragment lighting
+                    // PrimaryFragmentColor and SecondaryFragmentColor are both for fragment lighting
+                    // For now, we just make them the same as PrimaryColor.
                     case Source::PrimaryColor:
                     case Source::PrimaryFragmentColor:
+                    case Source::SecondaryFragmentColor:
```
Comment (neobrain) [inline src/video_core/rasterizer.cpp]: should return 0 instead.

**Comment by Subv** [comment]
LGTM

**Comment by neobrain** [comment]
Please prefix the commit message with `Pica/Rasterizer: `.

**Comment by MoochMcGee** [comment]
Ah, crap.

**Comment by neobrain** [comment]
Why was this closed?

**Comment by MoochMcGee** [comment]
GitHub automatically closed it when I tried to redo it. Also, infinite
commits somehow.

**Pull request closed without merging**